### ISSUE TITLE
I would like to get programmar running locally

### ISSUE THREAD

**Issue by colons** [Layerful/programmar#3]
I would like to get programmar running locally so I can fiddle with it. Your readme doesn't have any setup instructions, though; it's mostly just a changelog. There appear to be requirements listed for four different package managers in two different languages, but I'm a Python developer mostly so all this is \*very\* confusing. Which of them am I supposed to invoke, and in what order? Is there any configuration stuff I'd need to do first? Once that's done, how do I actually host it?

P.S. Out of curiosity, why do the commit messages all start with 'A - '?

P.P.S. I don't actually know what programmar is; I can't find any public explanations or evidence or artefacts and I mostly want to get it running locally so I can find out without giving people access to my GitHub email addresses.

**Comment by JosephSmith127** [comment]
Hey man, 

I've taken into consideration your issues, and why you'd like to see what it is before you allow us access to your github public information. Just to let you know the public information only includes the information found on your public profile, so if your email was private we couldn't collect that for example. Anyway, we've now opened up the platform for everyone. A basic explanation is Programmar is a place to read and write about development. You'll be able to access now and read some articles but you'll need to login before writing or enjoying articles.

As for the git commit messages, I find it easy to spot something as I go through as the A stands for "Append" so anything which is added.

A - append
D - delete
F - fix

Thats the basic run down.

As for installing it on your local, I use homestead vagrant and the framework it is in is Laravel, laravel.com has some great documentation on how you can set all that up. 

Thanks for your interest I hope this has made it more clear.

**Comment by colons** [comment]
Thanks! That… is probably enough information to get it up and running, but it'd be super cool if you provided an example setup procedure as something resembling a shell session; I've done `vagrant box add laravel/homestead` but I have no idea how to link that to this thing; there doesn't appear to be a vagrant config in this repo and I've not really used Vagrant before. I've also installed Lavarel locally, but if you're using Vagrant for local development it's probably sensible to do the same over here; I'm not sure what database I should be using or anything :<

Gosh that new homepage is so much better, thank you! Before that was a thing, the first thing I found by [searching twitter](https://twitter.com/search?q=programmar.io) was [this](http://programmar.io/article/1424685212), which probably isn't the best first impression I could have got.

**Comment by SimonJ** [comment]
I've would love to write a documentation how to set this up for local development.
Will it be okay @JosephSmith127 ?

**Comment by JosephSmith127** [comment]
Yes thats totally fine Simon! Thanks for your help! Refer to the Laravel documentation as much as possible, incase any changes happen to that also :)

**Comment by JosephSmith127** [comment]
Had little time, with a lot of layerful projects going on. So this is much appreciated.
